beacon_node/store/src/hot_cold_store.rs

```diff
@@ -211,6 +213,17 @@ impl<E: EthSpec> HotColdDB<E, LevelDB<E>, LevelDB<E>> {
             );
         }
 
+        if let Some(blob_info) = db.load_blob_info()? {
+            let oldest_blob_slot = blob_info.oldest_blob_slot;
+            *db.blob_info.write() = blob_info;
+
+            info!(
+                db.log,
+                "Blob info loaded from disk";
+                "oldest_blob_slot" => ?oldest_blob_slot,
+            );
+        }
```

Comment:
Lets put this _after_ the schema migration block below so that if we change the layout of `BlobInfo` in future we can apply a schema migration before trying to load it (which would error).